backend/src/services/v2/review.ts

```diff
@@ -132,9 +139,38 @@ export async function respondToReview(
   if (!update) {
     throw GenericError(500, `Adding response to Review was not successful`, { modelId, reviewIdQuery, role })
   }
+  sendReviewNotification(update, user)
   return update
 }
 
+export async function sendReviewNotification(review: ReviewDoc, user: UserDoc) {
```

Comment:
Could we change this to `sendReviewResponseNotification` please? A 'Review' is normally what we create when someone creates a release or an access request, here we wanting to send a notification when someone responds to that review.